src/battle/healthbar.c

```diff
@@ -1508,84 +1508,81 @@ static void DrawGauge (Healthbar * param0, u8 param1)
     }
 }
 
-static s32 UpdateGauge (s32 gaugeMax, s32 gaugeCur, s32 gaugeChange, s32 * gaugeTemp, u8 gaugeSize, u16 fillOffset)
+static s32 UpdateGauge (s32 max, s32 cur, s32 diff, s32 * temp, u8 size, u16 fillOffset)
 {
-    s32 gaugeUpdated;
-    s32 gaugeFinalValue;
-    u8 correctedSize;
-    s32 ratio;
+    s32 updated, final, ratio;
 
-    correctedSize = gaugeSize * 8;
+    u8 corrected = size * 8;
 
-    if (*gaugeTemp == -2147483648) {
-        if (gaugeMax < correctedSize) {
-            *gaugeTemp = gaugeCur << 8;
+    if (*temp == -2147483648) {
+        if (max < corrected) {
+            *temp = cur << 8;
         } else {
-            *gaugeTemp = gaugeCur;
+            *temp = cur;
         }
     }
 
-    gaugeUpdated = gaugeCur - gaugeChange;
+    updated = cur - diff;
 
-    if (gaugeUpdated < 0) {
-        gaugeUpdated = 0;
-    } else if (gaugeUpdated > gaugeMax) {
-        gaugeUpdated = gaugeMax;
+    if (updated < 0) {
+        updated = 0;
+    } else if (updated > max) {
+        updated = max;
     }
 
-    if (gaugeMax < correctedSize) {
-        if ((gaugeUpdated == (*gaugeTemp >> 8)) && ((*gaugeTemp & 0xff) == 0)) {
+    if (max < corrected) {
+        if ((updated == (*temp >> 8)) && ((*temp & 0xff) == 0)) {
```

Comment:
[nit] unnecessary parens. This is equivalent to:

```suggestion
        if (updated == (*temp >> 8) && (*temp & 0xff) == 0) {
```